libs/wlgen/wlgen/rta.py

```diff
@@ -133,7 +133,8 @@ def calibrate(target):
                 pload[cpu] = int(pload_match.group(1))
                 log.debug('>>> cpu%d: %d', cpu, pload[cpu])
 
-        # target.cpufreq.load_governors()
+        for cpu, governor in old_governors.iteritems():
+            target.cpufreq.set_governor('cpu{}'.format(cpu), governor)
```

Comment:
Possible suggestion: Iterating over all the CPUs can take quite a while. The cpufreq module has `get_domain_cpus` which exposes the frequency domains (cpufreq policies). If you set the governor for 1 CPU in a domain it's also set for all the other CPUs on the domain. So we can avoid some devlib calls.

Not really sure if it's worth the hassle, up to you guys.